docs/dev/tutorials/custom-aa-tutorial.md

```diff
@@ -36,23 +36,19 @@ yarn add -D typescript ts-node ethers@^5.7.2 zksync-web3 hardhat @matterlabs/har
 The current version of `zksync-web3` uses `ethers v5.7.x` as a peer dependency. An update compatible with `ethers v6.x.x` will be released soon.
 :::
 
-3. Add the zkSync and OpenZeppelin libaries.
 
-```sh
-yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable
```

Comment:
Why did you remove this line @krofax ?

Reply:
> Sorry, I checked the ticket realised why the code is added. All good apart from that line I mentioned.

What line?

Reply:
the one mentioned here ``````sh
yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable``` etc .. that's not all of it, but that's all i can see, the whole sh code block

Reply:
But you removed everything and Antonio is only asking for one to be removed?

Reply:
Updated.

Reply:
can you push the change please @krofax